fusillade/clouddirectory.py

```diff
@@ -0,0 +1,807 @@
+            if link_name.startswith('G'):
+                self._groups.append(unquote(link_name.split('->')[1]))
+            elif link_name.startswith('R'):
+                self._roles.append(unquote(link_name.split('->')[1]))
+
+    def _add_links(self, links: typing.List[str], link_type: str):
+        if not len(links):
+            return
+        parent_path = self.cd.get_obj_type_path(link_type)
+        operations = [self.cd.batch_attach_object(parent_path + link,
+                                                  self.object_reference,
+                                                  self._link_formats[link_type].format(parent=quote(link),
+                                                                                       child=self._path_name))
+                      for link in links]
+        self.cd.batch_write(operations)
+
+    def _remove_links(self, links: typing.List[str], link_type: str):
+        if not len(links):
+            return
+        parent_path = self.cd.get_obj_type_path(link_type)
+        operations = [self.cd.batch_detach_object(parent_path + link,
+                                                  self._link_formats[link_type].format(parent=quote(link),
+                                                                                       child=self._path_name))
+                      for link in links]
+        self.cd.batch_write(operations)
+
+    def lookup_policies(self) -> typing.List[str]:
+        return [policy['StringValue'] for policy in self.cd.lookup_policy(self.object_reference)]
+
+    @property
+    def name(self):
+        return self._name
+
+    @property
+    def policy(self):
+        if not self._policy:
+            self._policy = [i for i in self.cd.list_object_policies(self.object_reference)][0]
+        return self._policy
+
+    @property
+    def statement(self):
+        if not self._statement:
+            self._statement = self.cd.get_object_attrtibutes(self.policy,
+                                                             'IAMPolicy',
+                                                             ['Statement'])['Attributes'][0]['Value'].popitem()[1]
+        return self._statement
+
+    @statement.setter
+    def statement(self, statement: str):
+        params = [
+            UpdateObjectParams('IAMPolicy',
+                               'Statement',
+                               ValueTypes.StringValue,
+                               statement,
+                               UpdateActions.CREATE_OR_UPDATE)
+        ]
+        self.cd.update_object_attribute(self.policy, params)
+        self._statement = None
+
+    def _set_attributes(self, attributes: typing.List[str]):
+        resp = self.cd.get_object_attrtibutes(self.object_reference, self._object_type, attributes)
+        for attr in resp['Attributes']:
+            self.__setattr__('_' + attr['Key']['Name'], attr['Value'].popitem()[1])
```

Comment:
that's because of the format of the response. See the return response for [get_object_attribute](https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/clouddirectory.html#CloudDirectory.Client.get_object_attributes)